homeassistant/components/deconz/strings.json

```diff
@@ -11,11 +11,14 @@
         "data": {
           "host": "[%key:common::config_flow::data::host%]",
           "port": "[%key:common::config_flow::data::port%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of your deCONZ device."
```

Comment:
```suggestion
          "host": "The hostname or IP address of the device running deCONZ."
```

Reply:
Yes better!

Reply:
Or deCONZ host would perhaps be more appropriate

Reply:
you want to give it a go? I have the feeling you can come up with a better solution than me.

Reply:
Done